### DISCUSSION TITLE
fuse: failed to exec fusermount: Permission denied

### DISCUSSION THREAD

**Discussion by skeller80** [ImageMagick/ImageMagick#3436] [Development]
Hi, I am getting the following error when trying to get an images dimensions. 

Any help would be greatly appreciated.

Thanks,

```
magick identify -format "%[fx:w]x%[fx:h]" /mnt/MOUNTED_DRIVE/FILE.jpg
fuse: failed to exec fusermount: Permission denied

Cannot mount AppImage, please check your FUSE setup.
You might still be able to extract the contents of this AppImage
if you run it with the --appimage-extract option.
See https://github.com/AppImage/AppImageKit/wiki/FUSE
for more information
open dir error: No such file or directory
```

**Comment by fmw42** [comment]
This does not seem to be an imagemagick issue. It looks like your disk disk not mount. So you cannot get your image to find its dimensions What is your OS environment? What is your IM version.?

**Reply by skeller80** [reply]
I have actually just copied the file locally to make sure that it wasn't the mounted directory and am getting the same issue.

When I look at /usr/bin/fusermount it appears to be locked to the root user.

-rwsr-x---. 1 root fuse 32600 Feb 22 23:47 /usr/bin/fusermount

**Reply by skeller80** [reply]
Sorry, should have mentioned that I cant even get the version I get the same error. 

For OS.

cat /etc/os-release
NAME="Oracle Linux Server"
VERSION="7.9"
ID="ol"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.9"
PRETTY_NAME="Oracle Linux Server 7.9"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:oracle:linux:7:9:server"
HOME_URL="https://linux.oracle.com/"
BUG_REPORT_URL="https://bugzilla.oracle.com/"

ORACLE_BUGZILLA_PRODUCT="Oracle Linux 7"
ORACLE_BUGZILLA_PRODUCT_VERSION=7.9
ORACLE_SUPPORT_PRODUCT="Oracle Linux"
ORACLE_SUPPORT_PRODUCT_VERSION=7.9
